src/fundus/publishers/us/washington_post.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class WashingtonPostParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("div[data-qa='article-body'] > p, div[class='story relative'] > p")
+        _subheadline_selector = CSSSelector("h2[data-qa='subheadline']")
+
+        @attribute
+        def body(self) -> ArticleBody:
+            return extract_article_body_with_selector(
+                self.precomputed.doc,
+                paragraph_selector=self._paragraph_selector,
+                subheadline_selector=self._subheadline_selector,
+            )
+
+        @attribute
```

Comment:
You could consider also adding a function for topics, because within the json, theres a tag called keywords which would provide the necessary data